src/operators/soundtouch_op.ml

```diff
@@ -54,6 +59,9 @@ object (self)
 
   method stype       = source#stype
 
+  method self_sync   =
+    if rate = 1. then source#self_sync else false
```

Comment:
This does not even compile, because `rate` is a float getter...

Reply:
Ha. Then we can't do this, as the `rate` might change after the clock is set..

Reply:
Yes, let's just return `false` after all :)